docs/namespaced-broker-resources.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Using Namespaced Broker Resources
+layout: docwithnav
+---
+
+# Cluster-Scoped vs Namespace-Scoped Broker Resources
+
+Service Catalog enables service brokers to be registered in two manners: as a cluster-scoped resource or as a namespace-scoped resource. As a user of service catalog, you might use these approaches to accomplish different goals, like providing a common set of service broker resources to all users or extending your existing role based access control (RBAC) policies to the lifecycle of service broker resources. This document will explain some use cases for namespace-scoped resources and how to accomplish them using Service Catalog.
+
+## Possible Use Cases
+
+When using `ClusterServiceBroker` and associated `ClusterServiceClass` and `ClusterServicePlan` resources, service broker resources existing within the cluster as cluster-scoped resources. This means that you are limited in how you can apply RBAC and you can only have a single instance of that resource for a given identifier. As an example, if the service broker you are registering has fixed class and plan identifiers, you will be limited to one instance of the broker. With namespace-scoped brokers, however, the `ServiceBroker`, along with the `ServiceClass` and `ServicePlan` resources are scoped to a particular namespace. This allows for some more complicated use cases that were not possible with the cluster-scoped broker resources. 
+
+### Registering Brokers Per Namespace
+
+A service broker that provisions services in a cloud provider usually needs credentials in order to complete the request on behalf of Service Catalog. Some organizations may provide different access credentials to different teams in order to separate billing usage or to isolate control of resources. In these cases, the cluster operator might want to register two copies of the broker using different credentials for each team. When using `ClusterServiceBroker` and the associated `ClusterServiceClass` and `ClusterServicePlan` resources, it was not possible to register two instances of a service broker unless each registration could provide unique identifiers for service c lasses and service plans.
+
+Using namespace-scoped brokers, however, enables the broker to be installed in each namespace without conflicting at the class and plan level. When creating a service instance, you specify either the external class or plan name, or provide the class or plan identifier. Service Catalog then resolves these in order to determine which broker it should issue the provision command to. When using namespace-scoped brokers and their associated resources, this resolution occurs within the namespace. That means that users in namespace `backend-team` and namespace `front-end-team` can have the their own broker registrations and provision requests will be issued to the correct broker.
```

Comment:
nit:  make backend and frontend use the same hypen break (ie instead of *front-end-team*, use *frontend-team* to match your *backend-team*)